runtime/oti/jvminit.h

```diff
@@ -616,7 +616,6 @@ enum INIT_STAGE {
 #define SYSPROP_JDK_MODULE_PATCH "jdk.module.patch."
 #define SYSPROP_JDK_MODULE_ILLEGALACCESS "jdk.module.illegalAccess"
 #define JAVA_BASE_MODULE "java.base"
-#define JAVA_MODULE_VERSION "9"
```

Comment:
Would it be a more straightforward change to use the preprocessor to stringify the JAVA_SPEC_VERSION here?

https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html

Reply:
Modified to use `J9_STR(JAVA_SPEC_VERSION)`.